cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -542,6 +542,28 @@ rules:
   - create
   - update
   - delete
+- apiGroups:
```

Comment:
Whatever changes we make in common.yaml will also need to be made in two places:
- The [helm chart](https://github.com/rook/rook/tree/master/cluster/charts/rook-ceph/templates)
- The integration tests: [ceph_manifests.go](https://github.com/rook/rook/blob/master/tests/framework/installer/ceph_manifests.go)

Reply:
Please add the fields for managePodBudgets too when you change the CRD :)
```
           disruptionManagement:
              properties:
                managePodBudgets:
                  type: boolean
                osdMaintenanceTimeout:
                  type: integer
                manageMachineDisruptionBudgets:
                  type: boolean
```

Reply:
done